subsys/event_manager_proxy/event_manager_proxy.c

```diff
@@ -288,7 +289,11 @@ static int send_event_to_remote(struct emp_ipc_data *ipc, const struct app_event
 	memcpy(buffer, eh, sizeof(buffer));
 	remote_eh->type_id = remote_ev;
 
-	int ret = ipc_service_send(&ipc->ept, buffer, sizeof(buffer));
+	for (size_t cnt = CONFIG_EVENT_MANAGER_PROXY_SEND_RETRIES + 1; cnt > 0; --cnt) {
+		ret = ipc_service_send(&ipc->ept, buffer, sizeof(buffer));
+		if (ret >= 0)
```

Comment:
Maybe checkpatch allows to do it but i don't!
I believe we were obliged to always use {} brackets, even for single line if statement.

Reply:
really? I don't think so...

Reply:
You are right, i heard it int the past, not even sure if its still applicable. Checkpatch doesn't complain so my nack is an abuse.

Reply:
Fixed